apps/selflow-cli/selflow_gen_docs.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+	"os"
+	"strings"
+)
+
+func NewGenDocsCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "gen-docs",
+		Short: "Generate the documentation [Development purpose]",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := generateDocs(); err != nil {
+				if _, err = fmt.Fprintf(os.Stderr, "%v\n", err); err != nil {
+					panic(err)
+				}
+				os.Exit(1)
+			}
+		},
+	}
+}
+
+const fileHeadingFormat = "---\nslug: %s\ntitle: \"⌨ %s\"\n---\n\n# ⌨️ `%s`\n\n"
+
+func generateDocs() error {
+	return doc.GenMarkdownTreeCustom(rootCmd, "./docs/ecosystem/cli", func(filepath string) string {
```

Comment:
this line work on windows based file system ?

Reply:
Thanks! I missed that :penguin: !